tests/pkg_api/test_pkg.py

```diff
@@ -28,16 +42,80 @@ def test_add_statement(monkeypatch: pytest.MonkeyPatch, user_pkg: PKG) -> None:
                 "Stavanger", URI("https://dbpedia.org/page/Stavanger")
             ),
         ),
-        logging_data={"authoredBy": URI("http://example.com/testuser")},
+        logging_data={
+            "authoredBy": URI("http://example.com/testuser"),
+            "authoredOn": "2024-02-05T13:54:32",
+        },
+    )
+
+
+@pytest.fixture
+def statement_with_concept() -> PKGData:
+    """Returns a statement."""
```

Comment:
Why do we need this fixture? Wouldn't the one above be enough?

Reply:
I forgot to remove this fixture when dividing the PR.
It will be needed later to test that we can parse statements with concepts.